changes/517.bugfix.rst

```diff
@@ -0,0 +1 @@
+A workaround for python/cpython#81061 was conditionally removed when running under Python 3.9 or earlier.
```

Comment:
I think you mean something like this, right? (also added link to the CPython issue)

```suggestion
A workaround for `python/cpython#81061 <https://github.com/python/cpython/issues/81061>`__ is now conditionally applied only for the Python versions that require it (Python 3.9 and earlier).
```